src/winetricks

```diff
@@ -10938,6 +10938,28 @@ load_mfc70()
 
 #----------------------------------------------------------------
 
+w_metadata msaa dlls \
+    title="MS Active Accessibility (oleacc.dll, oleaccrc.dll, msaatext.dll)" \
+    publisher="Microsoft" \
+    year="2003" \
+    media="download" \
+    file1="../msaa/MSAA20_RDK.exe" \
```

Comment:
You can just use the filename directly, since the file is in the directory with the same name as the package.

Reply:
Fixed, thanks!